pkgs/development/tools/jsonnet-bundler/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "jsonnet-bundler";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "jsonnet-bundler";
+    repo = "jsonnet-bundler";
+    rev = "v${version}";
+    sha256 = "0pk6nf8r0wy7lnsnzyjd3vgq4b2kb3zl0xxn01ahpaqgmwpzajlk";
+  };
+
+  vendorSha256 = null;
+
+  doCheck = true;
```

Comment:
`doCheck` is true by default:
https://github.com/NixOS/nixpkgs/blob/3d04e9a779a3ca28230e0e8f3c4310264960e979/pkgs/development/go-modules/generic/default.nix#L211

Reply:
Thanks, I removed it.